### PR TITLE
Include sql_variant type for buffered queries

### PR DESCRIPTION
**Pull request by yitam**
Fixed the issue reported in #1079 and #51

### CODE REVIEW

File: test/functional/sqlsrv/srv_1079_sql_variant_buffered_queries.phpt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  ["Type"]=>
+  int(-150)
+  ["Size"]=>
+  int(10)
+  ["Precision"]=>
+  NULL
+  ["Scale"]=>
+  NULL
+  ["Nullable"]=>
+  int(1)
```
Comment (david-puglielli) [inline test/functional/sqlsrv/srv_1079_sql_variant_buffered_queries.phpt]: So sqlsrv_field_metadata does not return the `sqlsrv:decl_type` that identifies a sql_variant type?

Comment (yitam) [reply test/functional/sqlsrv/srv_1079_sql_variant_buffered_queries.phpt]: Yes, in the [sqlsrv_field_metadata documentation](https://docs.microsoft.com/sql/connect/php/sqlsrv-field-metadata?view=sql-server-ver15) the TYPE returns the numeric value of the sql type, and here ` ["Type"]=>  int(-150)`, which is `SQL_SS_VARIANT`

File: .travis.yml [hunk 2]
```diff
@@ -20,10 +20,10 @@ env:
   - TEST_PHP_SQL_PWD=Password123
 
 before_install:
-  - docker pull mcr.microsoft.com/mssql/server:2019-GA-ubuntu-16.04
+  - docker pull mcr.microsoft.com/mssql/server:2019-latest
```
Comment (david-puglielli) [inline .travis.yml]: This installs the ubuntu version of SQL Server?

Comment (yitam) [reply .travis.yml]: Yes, please scroll down this [page for your reference ](https://hub.docker.com/_/microsoft-mssql-server)

**Comment by Beakerboy** [comment]
Thanks for fixing it so quickly! What’s the typical timeline for rolling these fixes out to PECL? Which versions will it be applied to?

**Comment by yitam** [comment]
@Beakerboy consider yourself lucky this time ;) as this should be merged to the next stable release shortly

**Pull request merged**